src/main/java/io/mosip/digitalcard/DigitalCardApplication.java

```diff
@@ -0,0 +1,26 @@
+package io.mosip.digitalcard;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
```

Comment:
Entire unit test cases are missing, please add them.

Reply:
sure @vishwa-vyom working on that but deployment is required so I have not added it for now.

Reply:
ok, let not resolve this comment still that is taken care